### PR TITLE
Add catalog description

### LINKED ISSUE #4224: [Docs] Add Catalog project description to Layer5 ReadMe file.
### Current Behavior

Currently there is no description of the Catalog project on the Layer5  ReadMe file.

### Desired Situation

We need a breif summary of the Catalog Project along with the link to the project page and logo. 

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by goldfishdolphin**
**Description**

This PR fixes #4224 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by goldfishdolphin** [comment]
@leecalcote please have a look at the catalog description. I have changed the description slightly. 
I may need different links for <a>.

**Comment by l5io** [comment]
🚀 Preview for commit b85e25b287af866b8be73c996ef34375e4c4c74e at: https://6463912ec34afa255f9fea57--layer5.netlify.app

File: README.md [hunk 1]
```diff
@@ -133,6 +133,19 @@ alt="Nighthawk" align="left" />
 <br /><br /><br />
 </p>
 
+
+
+<p style="clear:both;">
+<h2><a href="https://layer5.io/projects/catalog">Meshery Catalog</a></h2>
+<a href="">
+  <img src=".github/assets/images/catalog/catalog.svg"
```
Comment (leecalcote) [inline README.md:141]: @goldfishdolphin does this same logo version work in both light and dark modes? I'm not suggesting that it does or doesn't. I'm just confirming.

File: README.md [hunk 2]
```diff
@@ -133,6 +133,19 @@ alt="Nighthawk" align="left" />
...
+style="float:left;margin:10px;" width="125px"
+alt="Meshery Catalog" align="left" />
+</a>
+<a href="">Meshery Catalog</a> is the ultimate solution for DevOps engineers looking to optimize their cloud native infrastructure configurations and operational patterns. As a central hub for sharing cloud native infrastructure designs, Meshery Catalog enables the exchange of the best practices, reusable templates, and Kubernetes-based operational patterns for multi-cluster Kubernetes clusters and distributed applications.
```
Comment (leecalcote) [inline README.md:145]:
```suggestion
<a href="">Meshery Catalog</a> As a central hub for sharing cloud native infrastructure designs, Meshery Catalog enables the exchange of the best practices, reusable templates, and Kubernetes-based operational patterns for multi-cluster Kubernetes clusters and distributed applications. Meshery Catalog serves as a marketplace full of curated cloud native infrastructure configurations and operational patterns perfect for DevOps engineers who seek to leverage and contribute to Meshery's large collection of design patterns.
```

File: README.md (hunk 1 above)
Comment (goldfishdolphin) [reply README.md:141]: It is a two-tone logo that works fine in dark and light mode. 
This image has been recommended by @leecalcote for the catalog card on the projects page.

Comment (leecalcote) [reply README.md:141]: Ok. 👌

**Pull request merged**